author: Davide Grassano (CECAM)
Use sources downloaded by EasyBuild instead of cloning from git
diff --git a/CMakeExternals/PythonQt.cmake b/CMakeExternals/PythonQt.cmake
index 3e190556..c1c78488 100644
--- a/CMakeExternals/PythonQt.cmake
+++ b/CMakeExternals/PythonQt.cmake
@@ -92,15 +92,6 @@ if(NOT DEFINED PYTHONQT_INSTALL_DIR)
   endif()
 
   set(location_args )
-  if(${proj}_URL)
-    set(location_args URL ${${proj}_URL})
-  elseif(${proj}_GIT_REPOSITORY)
-    set(location_args GIT_REPOSITORY ${${proj}_GIT_REPOSITORY}
-                      GIT_TAG ${revision_tag})
-  else()
-    set(location_args GIT_REPOSITORY "https://github.com/commontk/PythonQt.git"
-                      GIT_TAG ${revision_tag})
-  endif()
 
   ExternalProject_Add(${proj}
     ${${proj}_EXTERNAL_PROJECT_ARGS}
